### PR TITLE
desktop version of the portfolio project

### PR DESCRIPTION
**Pull request by ANTOINE1128**
on this branch I made the desktop version of the portfolio :
- add Images 
- add HTML elements to fit the desktop version
- add styles to fit the desktop version

### CODE REVIEW

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @ANTOINE1128 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -1,483 +1,744 @@
...
-  margin-top: 100px;
-  padding: 0 24px;
-  flex-direction: column;
+/* contact section */
+.contact {
+  background-color: #fff;
+  display: grid;
+  padding: 4.3rem 0 9.6rem;
+  grid-template-columns: 20.5rem;
+  justify-content: center;
```
Comment (Qoosim) [inline style.css:479]: - [ ] Great job implementing changes required by the previous reviewer. However, there're few things you need to work on before this milestone could meet its requirements. It is required to stick to the figma design as close as possible. This includes, font, typography, margin, padding among others. You should try as much as you can to produce a close replica of the figma design. Look at the red sqaure I used to box the two parts on the section. the width that your contact information covers must be the same with the figma file, likewise the form. There should not be too much obvious difference between your design and figma design. Kindly give this another shot. I believe you can do it. Let's try it one more time.

### Your Design
your-form

### Figma Design
temp-form

File: style.css [hunk 2]
```diff
@@ -1,483 +1,744 @@
...
+  font-family: 'InaiMathi', 'Gill Sans', sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 22px;
-  line-height: 43px;
-  margin: 25px;
+  font-size: 1.3rem;
+  line-height: 2.6rem;
+  padding-top: 1.5rem;
+  padding-bottom: 1.5rem;
```
Comment (Qoosim) [inline style.css:470]: - [ ] Great job positioning the technologies on your page to the center.  But the tech lists are left-aligned as we have them on the figma design. Kindly fix that too.

### Your Design
your-list

### Figma Design
temp-list

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @ANTOINE1128, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @ANTOINE1128, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**